gfxr_ext/decode/dive_pm4_capture.h

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2025 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef GFXRECON_DECODE_DIVE_PM4_CAPTURE_H
+#define GFXRECON_DECODE_DIVE_PM4_CAPTURE_H
+
+#include "util/defines.h"
+
+GFXRECON_BEGIN_NAMESPACE(gfxrecon)
+GFXRECON_BEGIN_NAMESPACE(decode)
+
+class DivePM4Capture
+{
+public:
+    using CaptureFunc = void (*)();
+
+    static DivePM4Capture& GetInstance()
+    {
+        static DivePM4Capture instance;
+        return instance;
+    }
+
+    void MaybeStartCapture();
```

Comment:
nit: i think `Try` is a more common prefix for functions that attempt to perform an action but might fail or not proceed for some reason. 

void TryStartCapture()
void TryStopCapture()